charts/ziti-router/templates/configmap.yaml

```diff
@@ -56,8 +56,10 @@ data:
       {{ include "ziti-router.fullname" . }}-identity &>/dev/null; then
       kubectl -n {{ .Release.Namespace }} delete secret \
         {{ include "ziti-router.fullname" . }}-identity
+      sleep 60
```

Comment:
sleep 60???

Reply:
how about a while/sleep/echo then just saying that you're harmlessly waiting for no reason so you can look at the logs if you wanna then?

but, is it _really_ needed? seems like it's just waiting for no good reason but i admin i don't actually understand what's going on... :D it's just strange to see a one minute wait.